cmd/worker/main.go

```diff
@@ -169,7 +169,7 @@ func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blo
 		return err
 	}
 
-	results, _, _, err := worker.RunDynamicAnalysis(pkg, dynamicSandboxOpts)
+	results, _, _, err := worker.RunDynamicAnalysis(pkg, dynamicSandboxOpts, false)
```

Comment:
Why not just re-declare the PkgManager instances for the common image, and have the `pkgecosyste.Manager(api.Ecosystem)` changed to chose between two different sets?

It would avoid plumbing though a boolean to make the choice in a different location.

Reply:
Okay sure

Reply:
Done